tests/tests_geomstats/test_distributions/test_brownian_motion.py

```diff
@@ -0,0 +1,75 @@
+import random
+
+import pytest
+from scipy.stats import normaltest, pearsonr
+
+import geomstats.backend as gs
+from geomstats.distributions.brownian_motion import BrownianMotion
+from geomstats.geometry.euclidean import Euclidean
+from geomstats.test.test_case import TestCase, autograd_and_torch_only
+
+
+@autograd_and_torch_only
+@pytest.mark.slow
+class TestBrownianMotion(TestCase):
```

Comment:
Unit-tests should be unit, ie we should have at least one unit test per function.

Missing `test_sample_path` and missing `test__step`

Reply:
Hi @ninamiolane, thank you for this clarification!  With regards to this comment, would the ideal structure be to have a test class `TestBrownianMotion`, and then have this class have two methods `test_sample_path` and `test__step`?

Reply:
I think tests are already very good here, @ninamiolane. Two reasons: 1) this computation is relatively slow, so it is better to not add too many redundant tests; 2) there's randomness involved (we are sampling from a normal), which mean we cannot do a "normal test" (the checks done by @dralston78 are pretty clever).